Platform/ElkhartlakeBoardPkg/BoardConfig.py

```diff
@@ -348,7 +349,7 @@ def GetImageLayout (self):
                     ]
                 ),
                 ('REDUNDANT_B.bin', [
-                    ('UCODE.bin'    ,  ''        , self.UCODE_SIZE,    STITCH_OPS.MODE_FILE_PAD, STITCH_OPS.MODE_POS_TAIL),
+                    #('UCODE.bin'    ,  ''        , self.UCODE_SIZE,    STITCH_OPS.MODE_FILE_PAD, STITCH_OPS.MODE_POS_TAIL),

```

Comment:
('UCODE.bin' if self.HAVE_FSP_BIN != 0 else 'EMPTY',  ''        , self.UCODE_SIZE,    STITCH_OPS.MODE_FILE_PAD, STITCH_OPS.MODE_POS_TAIL),

Reply:
We can also use special flags to remove a component. 
ucode_flag = 0 if self.HAVE_FSP_BIN else STITCH_OPS.MODE_FILE_IGNOR
Then change:  "STITCH_OPS.MODE_FILE_PAD"  to  "STITCH_OPS.MODE_FILE_PAD | ucode_flag" for ucode entry.

Reply:
It looks much better rather than changing component name. Thanks Maurice.

Reply:
This is much smarter way then giving dummy fsp, ucode and vbt binaries initially thought. Thanks @myguitar !

Reply:
Done, thank you for the review :)